src/collapsible.tab.js

```diff
@@ -100,7 +100,7 @@
 
 			if( !self.collapsed ) {
 				activateTab( self.$tabHeader );
-				self._expand();
+				self.expand();
```

Comment:
This method call changed but I don't see it being renamed anywhere in the diff?

Reply:
I decided to use the public one and allow expand events to fire. Seemed more useful